src/test/codegen/intrinsics/is_const_eval.rs

```diff
@@ -0,0 +1,13 @@
+// compile-flags: -C no-prepopulate-passes
+
+#![crate_type = "lib"]
+#![feature(core_intrinsics)]
+
+use std::intrinsics::is_const_eval;
+
+// CHECK-LABEL: @is_const_eval
+#[no_mangle]
+pub unsafe fn is_const_eval() -> bool {
+    // CHECK: sdiv exact
```

Comment:
I have no idea what this does.. explain please

Reply:
This should error with the codegen that the intrinsic generates, which would be hopefully something like `ret %n` where `%n = i1 0` or similar.

Reply:
Oh... you're waiting.g for CI to tell you? Wouldn't it have been faster to run that test locally?

Reply:
It would, if LLVM wouldn't have changed in the last nightly. It literally has taken 4 hours to rebuild everything.

Reply:
Stage 1 should suffice and you can just run your tests, no others should break. That should speed up things